webui/react/src/components/NavigationTabbar.tsx

```diff
@@ -79,6 +85,19 @@ const NavigationTabbar: React.FC = () => {
       </div>
       <ActionSheet
         actions={[
+          {
+            render: () => {
+              return <AvatarCard className={css.user} user={auth.user} />;
+            },
+          },
+          {
+            label: 'Settings',
+            onClick: () => openUserSettingsModal(),
+          },
+          {
+            label: 'Sign out',
+            path: paths.logout(),
```

Comment:
`nit` I noticed when I sign out using the overflow menu, and sign back in, the overflow menu is disabled until I refresh the page. pretty niche flow but I thought I'd mention.

Reply:
It might be that wrapping with `handlePathUpdate` like with cluster logs handles this